tests/cli/test_jobs.py

```diff
@@ -19,3 +19,4 @@ def test_list_jobs(CLIENT, runner):
     time.sleep(0.5)
     assert result.exception is None
     assert result.exit_code == 0
+    CLIENT.delete_dataset(dataset.id)
```

Comment:
Can we refactor this test to use a fixture or setup/teardown? When an assertion fails, we won't delete the dataset otherwise.

Reply:
Yes, of course. Lazy monday thinking 🙂